docs/api-ref.md

```diff
@@ -3188,7 +3227,8 @@ Attribute | Description
 `name` | The name of the site. The name of the default site is "".
 `content_url` | The path to the site.
 `admin_mode` | (Optional) For Tableau Server only. Specify `ContentAndUsers` to allow site administrators to use the server interface and **tabcmd** commands to add and remove users. (Specifying this option does not give site administrators permissions to manage users using the REST API.) Specify `ContentOnly` to prevent site administrators from adding or removing users. (Server administrators can always add or remove users.)
-`user_quota`| (Optional) Specifies the maximum number of users for the site. If you do not specify this value, the limit depends on the type of licensing configured for the server. For user-based license, the maximum number of users is set by the license. For core-based licensing, there is no limit to the number of users. If you specify a maximum value, only licensed users are counted and server administrators are excluded. Mutually exclusive with tiered license level settings.
+`user_quota`| (Optional) Specifies the total number of users for the site. The number can't exceed the number of licenses activated for the site; and if tiered capacity attributes are set, then `user_quota` will equal the sum of the tiered capacity values, and attempting to set `user_quota` will cause an error. 
+Tiered capacity attributes: `tier_explorer_capacitytier_explorer_capacity` `tier_creator_capacitytier_creator_capacity` `tier_viewer_capacity`| (Optional) The maximum number of licenses for users with the Creator, Explorer, or Viewer role, respectively, allowed on a site.  
```

Comment:
I think you still need to fix these two names: `tier_explorer_capacitytier_explorer_capacity` `tier_creator_capacitytier_creator_capacity

Reply:
It should be fixed now.  Thank you!